src/adafruit_circuitplayground/express.py

```diff
@@ -37,7 +37,7 @@ def __init__(self):
             (0, 0, 0),
         ]
         self.__state[CONSTANTS.EXPRESS_STATE.RED_LED] = False
-        self.__state[CONSTANTS.EXPRESS_STATE.SWITCH] = False
+        self.__state[CONSTANTS.EXPRESS_STATE.SWITCH] = True
```

Comment:
Is the switch supposed to be on by default?

Reply:
You're right. After looking at the pictures, it seems that default is off.